spec/mailers/user_mailer_spec.rb

```diff
@@ -10,12 +10,12 @@
       expect(welcome_email.from).to eql(['contact@theodinproject.com'])
     end
 
-    it 'includes an attachment' do
-      expect(welcome_email.attachments.count).to eq(1)
+    it 'includes a image' do
+      expect(welcome_email).to have_css 'img'
     end
 
     it 'renders the correct attachment' do
```

Comment:
Nit: We can update this description to better match what the updated test is doing

```suggestion
    it 'renders The Odin Project logo' do
```